### ISSUE TITLE
Plugin GUIs are too narrow

### ISSUE THREAD

**Issue by drautzburg** [muse-sequencer/muse#571]
Small nuisance here:

Whenever I open the controls for an audio plugin, I find the GUI too narrow. I can see the labels and the input fields okay, but the sliders are only about 1cm (about 3/8 inches)  wide. I always have to resize the window in order to use the sliders.

After resizing and closing the window, the new size is forgotten and next time it comes up narrow again. I could not find a way to make the wider windows permanent.

**Comment by terminator356** [comment]
Ahem... Sorry for the late reply. Been saying that a lot lately...
I fixed all the generic GUIs. Stay tooned, will try to fix LV2 and MESS
 positions and sizes.

Back in the Qt3 days I could swear the geometry used to automatically be remembered
 between innocently simple hide() and show() calls. Does anyone remember?
If true, well, not anymore. From Qt4 on there's a help clause about only QDialog
 automatically remembering its geometry. Thus we had no code (or need) to purposely
 remember and restore the GUI geometry between hide() and show() calls. I thought maybe
 we did, but maybe it was just broken. But upon checking we had no such code.
We should not need such code in my opinion. It might be X11's fault. Still, it's really very stupid.
So I added it now. But it kept acting weird, flaky, wouldn't keep my position.
So I found what I am certain is a Qt bug and reported it... yet another upstream bug.
I wrote a workaround. 
Try 'er out. Tim.

**Comment by drautzburg** [comment]
This is perfect now.

However I noticed, that adding a plugin to a track is not consided a "change", i.e. I can exit MusE without being asked to save the project.

**Comment by terminator356** [comment]
Hi, further to the fix a few days ago for the generic plugin GUIs:
Fixed LV2 synths position/size.

NOTE: AFAIK we have no control over LV2 synth plugin 'external'
 windows, only the ones where /we/ create the window.

TODO: Same fix for MESS synths...

There's a bit of redundancy with Andrew setting the size of the
 LV2 windows on creation, but I took ultimate control over it 
 during hide() and show() events to guarantee that the saved
 geometry is honoured.

Try 'er out. If you notice anything unusual, let me know.

Tim.
